src/transfer.ts

```diff
@@ -74,3 +74,53 @@ export async function ensureFunded(funding: EnsureFundedParams, algod: Algodv2,
 
   return undefined
 }
+
+interface TransferAssetParams extends SendTransactionParams {
+  senderAccount: SendTransactionFrom
+  receiverAccount: Account
+  assetId: number
+  amount: number | bigint
```

Comment:
I really like how `transferAlgos` types `amount` as `AlgoAmount`. It makes it super clear when your dealing with Algos or MicroAlgos and allows users to seamlessly use either.

Obviously transferAsset is a bit more complex, as we don't know what we are dealing with or the dc value for the asset.
I think we should make sure the docs are clear that we expect the smallest unit value, rather than the nominal unit value.